Sources/Context.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-public func enrich(context: [AnyHashable: Any], parameters: [String]) throws -> [AnyHashable: Any] {
+public func enrich(context: [String: Any], parameters: [String]) throws -> [String: Any] {
```

Comment:
add `env` as a parameter to `enrich`, with default value `ProcessInfo().environment`, both for testability and in case other apps/clients of the framework want to alter the env before enriching (to hide some sensible env vars for example)